roles/wordpress/files/kielipankki/corpora.php

```diff
@@ -29,6 +31,18 @@
   }
 }
 
+if ($lang=='fi') {
+    $lang_locale = 'lang="fi"';
+}
+
+if ($lang=='en') {
+    $lang_locale = 'lang="en-GB"';
```

Comment:
Did we ever ask the UH people about the preferred flavo(u)r of English?

Reply:
Anyway, if this is the preferred language code, I think this PR is all :+1: (and if not, then just change this and it is).